src/middleware/withDelegationStubs.js

```diff
@@ -53,21 +54,7 @@ export const withDelegationStubs = (handler) => async (request, env, ctx) => {
     delegationProofs: [], // Delegation proofs are set by withAuthorizedSpace handler
     locator:
       stubSpace && isDIDKey(stubSpace)
-        ? {
-            locate: async (digest, options) => {
-              const locateResult = await ctx.locator.locate(digest, options)
-              if (locateResult.error) return locateResult
-              return {
-                ok: {
-                  ...locateResult.ok,
-                  site: locateResult.ok.site.map((site) => ({
-                    ...site,
-                    space: stubSpace
-                  }))
-                }
-              }
-            }
-          }
+        ? spaceScopedLocator(ctx.locator, [stubSpace])
```

Comment:
What is a `stubSpace`?

Reply:
Gotcha - for testing right?

Reply:
Exactly—that's what this middleware is for.